### ISSUE TITLE
bank holiday´s with half day value

### ISSUE THREAD

**Issue by kamilkosek** [timeoff-management/application#223]
Hi, 
sorry to annoy you, but is it possible to add a bank holiday with a half day value ? 
In our company it´s usual to take 0,5 days on 24th December and 0,5 days on 31st of December

**Comment by vpp** [comment]
Hi Kamil,

Bank holidays are full days in current implementation.

Although the case you've described is interesting and worth thinking about.

**Comment by kamilkosek** [comment]
Yeah, I think it is common in germany to take these day´s as half day holidays.

**Comment by vpp** [comment]
To add such type of bank holidays is not hard. More trickier would be to integrate that into leave overlap checking logic.

Let's keep that ticket open for awhile and see what community thinks about that idea.

**Comment by kamilkosek** [comment]
Btw, do you have any hints where I could start implementing it ? I´m not a web dev but have some experience in .net languages so maybe it will work out :P

**Comment by vpp** [comment]
Kamil, I do not have access to desktop at this moment to send you any links..

**Comment by philipthomasme** [comment]
Definitely needed in Germany - I did not take a look at the code yet to see how much work it would be.

**Comment by mariobento** [comment]
+1 for Portugal, would be awesome to have this feature!

**Comment by elray** [comment]
+! for UK. We would like to see half days as well
